languages/en.ini

```diff
@@ -773,6 +773,7 @@ Preferred Library = "Preferred Library"
 Prev = "Prev"
 Preview = "Preview"
 Preview from = "Preview from"
+Previous = "Previous"
```

Comment:
Do we need a separate Previous translation when we already have Prev? If using an unabbreviated string is preferable for accessibility, that's fine with me -- I just want to be sure we're doing this intentionally.